README.md

```diff
@@ -0,0 +1,64 @@
+# BoutrosLab.plotting.general
+
+BoutrosLab.plotting.general (BPG) is a software package for generating publication-quality, customizable plots. 
+
+## Description
+
+Contains several plotting functions such as barplots, scatterplots, heatmaps, as well as functions to combine plots and assist in the creation of these plots. 
+These functions will give users great ease of use and customization options in broad use for biomedical applications, as well as general purpose plotting. 
+Each of the functions also provides valid default settings to make plotting data more efficient and producing high quality plots with standard colour schemes simpler. 
+All functions within this package are capable of producing plots that are of the quality to be presented in scientific publications and journals. 
+P'ng et al.; [BPG: Seamless, automated and interactive visualization of scientific data](doi:10.1186/s12859-019-2610-2); BMC Bioinformatics 2019.
+
+## Included plots
+![](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12859-019-2610-2/MediaObjects/12859_2019_2610_Fig1_HTML.png?as=webp)
+
+Available chart-types. The basic chart-types available in BPG: **a** density plot, **b** boxplot, **c** violin plot, **d** segplot, **e** strip plot, **f** barplot, **g** scatterplot, **h** histogram, **i** qqplot fit, **j** qqplot comparison, **k** Manhattan plot, **l** polygon plot, **m** heatmap, **n** dotmap and **o** hexbinplot.
+
+![](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12859-019-2610-2/MediaObjects/12859_2019_2610_Fig2_HTML.png?as=webp)
+
+Multiplot example. BPG's create.multiplot function is able to combine multiple chart-types together into a single figure as shown above.
+
+## Installation
+To install the latest public release of BoutrosLab.plotting.general from CRAN:
+
+```
+install.packages("Boutros.plotting.general")
+```
+
+Or to install the latest development version from Github:
+```
+# install.packages("devtools")
+
+devtools::install_github("uclahs-cds/public-R-BoutrosLab-plotting-general")
+```
+
+## Resources
+Available resources on BPG usage can be found at the package [CRAN page](https://cloud.r-project.org/web/packages/BoutrosLab.plotting.general/index.html), [reference manual](https://cloud.r-project.org/web/packages/BoutrosLab.plotting.general/BoutrosLab.plotting.general.pdf), or [vignette](https://cloud.r-project.org/web/packages/BoutrosLab.plotting.general/vignettes/PlottingGuide.pdf).
```

Comment:
We should add Function reference as well https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/reference/index.html